redis/src/commands/mod.rs

```diff
@@ -3317,22 +3335,32 @@ impl<Db: ToString> ToSingleRedisArg for CopyOptions<Db> {}
 ///
 /// # Example
 /// ```rust,no_run
-/// use redis::{Commands, RedisResult, SetOptions, SetExpiry, ExistenceCheck};
+/// use redis::{Commands, RedisResult, SetOptions, SetExpiry, ExistenceCheck, ValueComparison};
 /// fn set_key_value(
 ///     con: &mut redis::Connection,
 ///     key: &str,
 ///     value: &str,
 /// ) -> RedisResult<Vec<usize>> {
 ///     let opts = SetOptions::default()
 ///         .conditional_set(ExistenceCheck::NX)
+///         .value_comparison(ValueComparison::IFEQ("old_value".to_string()))
 ///         .get(true)
 ///         .with_expiration(SetExpiry::EX(60));
 ///     con.set_options(key, value, opts)
 /// }
 /// ```
-#[derive(Clone, Copy, Default)]
+#[derive(Clone, Default)]
 pub struct SetOptions {
     conditional_set: Option<ExistenceCheck>,
+    /// IFEQ `match-value` - Set the key's value and expiration only if its current value is equal to `match-value`.
```

Comment:
since these are new, please add to the docs from which version they're available, and phrase it in a way that clarifies that it's only on Redis.